form/bootstrap5.rst

```diff
@@ -187,7 +187,7 @@ class to the ``row_attr`` option.
 
     .. code-block:: php
 
-        $builder->add('email', CheckboxType::class, [
+        $builder->add('email', TextType::class, [
```

Comment:
```suggestion
        $builder->add('email', EmailType::class, [
```
Using the `EmailType` could be even better!

Reply:
Done while merging. Thanks!